src/lumigo_tracer/utils.py

```diff
@@ -161,3 +162,62 @@ def is_aws_environment():
     :return: heuristically determine rather we're running on an aws environment.
     """
     return bool(os.environ.get("LAMBDA_RUNTIME_DIR"))
+
+
+class Frame:
+    MAX_VAR_LEN = 200
+
+    total_frames_size: int = 0
+
+    def __init__(self, frame_info: inspect.FrameInfo):
+        self.lineno: int = frame_info.lineno
+        self.file_name: str = frame_info.filename
+        self.function: str = frame_info.function
+        Frame.total_frames_size += len(str(self.lineno)) + len(self.file_name) + len(self.function)
+        self.variables: FrameVariables = Frame._truncate_locals(frame_info.frame.f_locals)
+
+    def to_dict(self) -> Dict[str, Union[int, str, FrameVariables]]:
+        return {
+            "lineno": self.lineno,
```

Comment:
change it to be non-pythonic (the rest of the tracer should have the same interface)